lib/main.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:mood_tracker/app.dart';
 
-void main() {
+void main() async {
```

Comment:
please remove async, it is not needed here (you don't have async functions)

Reply:
removed